TestProjects/NotificationSamples/Assets/Scripts/AndroidTest.cs

```diff
@@ -0,0 +1,262 @@
+﻿using System;
+using System.Reflection;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using UnityEngine;
+using UnityEngine.UI;
+
+#if PLATFORM_ANDROID
+using Unity.Notifications.Android;
+#endif
+
+public class AndroidTest : MonoBehaviour
+{
+    public List<AndroidNotificationTemplate> androidNotifications;
+
+#if PLATFORM_ANDROID
```

Comment:
wondering, why are you wrapping this code with Android, this project is referencing notification packages, so all the API you use here, is also available in Editor. So why wrap it?

Reply:
I added "|| UNITY_EDITOR" to these ifdefs in latest commit, because it doesn't compile if I remove them completely - e.g. when building on Android it tries to compile files that reference iOS namespace